src/tokens/themes/red.css

```diff
@@ -0,0 +1,73 @@
+.theme-red {
+  --color-hover-secondary: var(--gray800);
```

Comment:
metterei un po' di ordine nei token, raggruppandoli per categoria (primary, secondary, etc...) e magari mettendo un commento che descrive la categoria in cima a ogni gruppo. lo so che negli altri non c'è ma secondo me andrebbe fatto anche lì.

i token dei link sono universali quindi uguali per tutti i temi, non dovrebbe servire metterli anche qui (ho visto che ci sono già nel tema default e quindi già definiti quando si importa il DS). idem per quelli degli status credo (success, warning, error)